.travis.yml

```diff
@@ -101,6 +107,17 @@ before_script:
     if [[ "$TRAVIS_PYTHON_VERSION" != "3.3" ]]; then
       python -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
     fi
+  # Only keep non-Python bears on Python 3.4
+  - >
+    if [[ "$BEARS" != "all" ]]; then
+      rm $(comm -23 <(ls bears/*/[A-Za-z]*.py | grep -v general | sort) <(grep -l PipRequirement bears/*/*.py | sort))
+      rm bears/general/CPDBear.py
+      rm bears/generate_package.py tests/generate_packageTest.py
```

Comment:
the special case for `bears/generate_package.py` -> `tests/generate_packageTest.py` could be added to `tests.prune.sh`, instead of here in .travis.yml